tiledb/__init__.py

```diff
@@ -114,3 +115,17 @@
 #
 # Note: 'pip -e' in particular will not work without this declaration:
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+try:
```

Comment:
Could you add a comment here explaining why we're doing this? (e.g. "If tiledb.cloud is installed, add CloudArray methods to TileDB arrays")

Reply:
:heavy_check_mark: